static/app/views/performance/transactionSummary/transactionSpans/spanDetails/exclusiveTimeTimeSeries.tsx

```diff
@@ -146,7 +147,8 @@ export default function ExclusiveTimeTimeSeries(props: Props) {
                   axisLabel: {
                     color: theme.chartLabel,
                     // p50() coerces the axis to be time based
-                    formatter: (value: number) => axisLabelFormatter(value, 'p50()'),
+                    formatter: (value: number) =>
+                      axisLabelFormatter(value, aggregateOutputType('p50()')),
```

Comment:
```suggestion
                      axisLabelFormatter(value, aggregateOutputType('duration')),
```
Right? And the comment above is not needed

Reply:
Yeah exactly! except it would be
`(value, 'duration')`